.travis.yml

```diff
@@ -0,0 +1,18 @@
+branches:
+  only:
+    - master
+cache:
+  directories:
+    - vendor
```

Comment:
According to examples in Travis' own documentation, this is sufficient.  Also keep in mind that I'm trying to not _be_ a "go project" since I'm using a fully containerized dev/build/test environment.  Outside the container, `$GOPATH` isn't a thing for me.